frontend/test/pytest/test_template.py

```diff
@@ -251,24 +302,24 @@ def ansatz(weights):
 
     layers = 2
     shape = qml.GateFabric.shape(n_layers=layers, n_wires=qubits)
-    params = jnp.array(np.random.random(size=shape))
+    params = jnp.array(np.ones(shape))
 
     interpreted_fn = qml.QNode(ansatz, device)
     jitted_fn = qjit(interpreted_fn)
+
     assert np.allclose(interpreted_fn(qml.numpy.array(params)), jitted_fn(params))
 
 
 def test_uccsd(backend):
     """Test UCCSD."""
 
     symbols = ["H", "H", "H"]
-    geometry = pnp.array(
+    geometry = np.array(
```

Comment:
Why this is changed?

Reply:
This was the only test using `pnp` so I switched to `np` for consistency.